it/uk/gov/hmrc/fileupload/WithITApplicationComponents.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.fileupload
+
+import org.scalatest.Suite
+import org.scalatestplus.play.OneServerPerSuite
+import play.api._
+import uk.gov.hmrc.mongo.MongoSpecSupport
+
+trait WithITApplicationComponents extends OneServerPerSuite with MongoSpecSupport {
```

Comment:
Do we need With prefixed to the name, IMO this was a bad naming convention used on another trait that shouldn't be copied